src/api/APIFactory.js

```diff
@@ -46,6 +47,11 @@ class APIFactory {
      */
     chunkedUploadAPI: ChunkedUploadAPI;
 
+    /**
+     * @property {MultiputUploadAPI}
+     */
+    multiputUploadAPI: MultiputUploadAPI;
```

Comment:
I don't think we use the word 'multiput' in any public documentation.  Also, it is confusing at first glance that we have both a chunked upload API and a multiput API.

Reply:
yeah I'll rename everything to chunked in a later PR.
Removing this line and related lines from this file for now.